test/utils/adjustMarketPrice.spec.ts

```diff
@@ -0,0 +1,194 @@
+import { checkMarketAndSmartAdjust } from 'hooks/usePrioritiseTokensForPrice'
+import tokenList from '../data/tokenList'
+
+// QUOTE gives price
+//     BASE/QUOTE
+// e.g WETH/TUSD
+
+describe('Check market and check base/quote token priorities', () => {
+  const [WETH, TUSD, USDT, , , , , FAKE] = tokenList
+
+  test('STABLE/STABLE => return with no change', () => {
+    // QUOTE
+    const sellToken = TUSD
```

Comment:
Why don't you define the constants for the token `USDT`, `TUSD` , so they have already the priority (instead of change in it in every test)

I think, if you do this, the tests become a bit shorter, easier to read, but more importantly less error prone, since you cannot mess up with priorities. 
We can use prio 1 for TUSD and USDT, 2 for EURO, 3 for WETH, ... (and do not change it)

Also, you can refactor the test to be onliners if you build a function:

```ts
 assertMarket({ TUSD, USDT  },  {  baseToken: TUSD, quote: USDT }) 
```

Reply:
i like doing extra work, obviously